sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -257,31 +257,37 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         """.stripMargin)
 
       // LOAD DATA INTO partitioned table must specify partition
-      withInputFile { path =>
+      withInputFile { f =>
         intercept[AnalysisException] {
+          val path = f.toURI.toString
```

Comment:
The variable is fine but can it be declared once before line 260? instead of on 262, 267, 271, 275